brave-core/src/main/java/com/github/kristofa/brave/BraveContext.java

```diff
@@ -0,0 +1,68 @@
+package com.github.kristofa.brave;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * The {@link BraveContext} is a registry of all {@link Brave} instances that have been created.
```

Comment:
would definitely like way to unregister specific `Brave` instance (e.g. as part of service lifecycle hook).